designs/extensible-cli-and-scaffolding-plugins-phase-1.md

```diff
@@ -170,8 +170,14 @@ Example Operator SDK main.go:
 
 ```go
 func main() {
+    rootCmdCfg := cli.RootCommandConfig{
+        CommandName: "operator-sdk",
+        //Short: "Short help text for operator-sdk",
+        //Long: "Longer help text for operator-sdk",
+        //Example: "Example usage of operator-sdk",
```

Comment:
Why we need to keep these commennts?

Reply:
I kept them to show an example usage of the new WithRootCommandConfig option.

Reply:
I think the example is ok. It is in the `designs` folder, not in the `pkg` one.